scripts/jenkins/docs.groovy

```diff
@@ -25,6 +25,7 @@ stage('Reports (Docs)') {
         keepAll: false, reportDir: 'build/docs', reportFiles: 'index.html',
         reportName: 'Doxygen'])
     step([$class: 'WarningsPublisher', canResolveRelativePaths: false,
+        messagesPattern: '.*ogs_file_.*,.*potential recursive class relation.*',
```

Comment:
They will become active end of this week, when ogs_file_param are all documented.